WORKSPACE

```diff
@@ -46,3 +46,31 @@ load(
 )
 
 install_rules_ros2_resolver_deps()
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
```

Comment:
This is nothing that we strictly need in the global `WORKSPACE` file, as it is only used to test cross compilation. I'd prefer adding that to the `examples` workspace or even create a new `WORKSPACE` for this.

@mvukov What do you think?

Reply:
Hi! I suggest it is run through CI anyways, to ensure compilation towards aarch64 doesn't break.

Reply:
Yes, I'm all for running it in CI, just setup a bit differently. We already have to jobs, one for the main workspace and one for examples. Would be easy to add a third. To trigger aarch64 compilation you need an extra job anyway.